e2e/Tests/iothub/service/FileUploadNotificationE2ETest.cs

```diff
@@ -41,12 +41,14 @@ public class FileUploadNotificationE2ETest : E2EMsTestBase
         [DataRow(IotHubTransportProtocol.WebSocket, 1, true)]
         public async Task FileUploadNotification_FileUploadNotificationProcessor_ReceivesNotifications(IotHubTransportProtocol protocol, int filesToUpload, bool shouldReconnect)
         {
+            // arrange
+
             var options = new IotHubServiceClientOptions
             {
                 Protocol = protocol
             };
 
-            using var cts = new CancellationTokenSource(TimeSpan.FromMilliseconds(TestTimeoutMilliseconds));
+            using var cts = new CancellationTokenSource(TimeSpan.FromSeconds(30));
```

Comment:
I think we can be more aggressive about this test because it shouldn't take so long to fail if the notification is not coming.

Reply:
Makes sense to me.